packages/app/lib/src/features/home/presentation/home/post.dart

```diff
@@ -204,7 +204,9 @@ class _PostImages extends ConsumerWidget {
               ),
             ),
             AsyncError() => const Text('Error loading image'),
-            _ => const CircularProgressIndicator(),
+            _ => const CircularProgressIndicator(
+              constraints: BoxConstraints(minWidth: 300, minHeight: 300),
```

Comment:
Should this be
```suggestion
              constraints: BoxConstraints.tight(Size.square(300)),
```

IDK, I don't want to _too_ big.

Reply:
Yeah, that looks better. I just wanted to change it so it wasn't weird and vertical anymore